packages/eds-core-react/src/components/Button/tokens/button.ts

```diff
@@ -98,7 +105,11 @@ export const button: ButtonToken = {
   },
   modes: {
     compact: {
-      height: compactButtonHeight,
+      minHeight: '1.7142em', // 24px
```

Comment:
Perhaps this is superfluous 🤔  Just wanted to see if it had any effect on layout shift.